ReactWindows/ReactNative.Shared/Bridge/ReactBridge.cs

```diff
@@ -106,13 +110,32 @@ public void Dispose()
             _jsExecutor.Dispose();
         }
 
-        private void ProcessResponse(JToken response)
+        private void ProcessResponse(JToken response, bool isEndOfBatch = false)
```

Comment:
What's the point of this argument?

Reply:
Seems like it might be in preparation for the `nativeFlushQueueImmediate` behavior. If that's the case, please defer these changes until that feature is pulled in as well. Ideally, that feature should also be done in a separate PR.

Reply:
I did try to do them separately before, but the changes were fairly intertwined. I can bring in nativeFlushQueueImmediate as well and rename this PR, if you like. lmk.

Reply:
Seems like that `isEndOfBatch` parameter is the only relic of the `nativeFlushQueueImmediate` feature (that I can see).